forms/user/registration_form.go

```diff
@@ -0,0 +1,71 @@
+package forms
+
+import (
+	"go-crawler-challenge/helpers"
+	"go-crawler-challenge/models"
+
+	"github.com/beego/beego/v2/core/validation"
+)
+
+type RegistrationForm struct {
+	Email           string `valid:"Required; MaxSize(100)"`
+	Password        string `valid:"Required; MinSize(6); MaxSize(12)"`
+	ConfirmPassword string `valid:"Required; MinSize(6); MaxSize(12)"`
+}
+
+var ValidationMessages = map[string]string{
+	"ExistedEmail":      "Email is already in use",
```

Comment:
Minor ENglish syntax issue:

```suggestion
	"ExistingEmail":      "Email is already in use",
```

Reply:
Updated here https://github.com/Lahphim/go-crawler-challenge/pull/36/commits/d0ac5bc174e99f21819ea012ab3cfeb833c77c82